src/Sulu/Bundle/WebsiteBundle/Controller/AnalyticsController.php

```diff
@@ -99,13 +96,12 @@ public function getAction($webspace, $id)
     /**
      * Creates a analytics for given webspace.
      *
-     * @param string $webspace
-     *
      * @return Response
      */
-    public function postAction(Request $request, $webspace)
+    public function postAction(Request $request)
     {
         $data = $request->request->all();
+        $webspace = $request->attributes->getString('webspace');
```

Comment:
doesnt make sense to change this to attributes->getString from my point of view. We should if possible keep the `, $webspace`

Reply:
Yes but this will put it into the route path again which we don't want.

Reply:
we should skip the `type: rest`

Reply:
the API urls should stay the same as before

Reply:
Routing changes have been removed as this was handled in your PRs.